fastlane/Fastfile

```diff
@@ -35,6 +37,18 @@ lane :bump do |options|
   push_to_git_remote(set_upstream: true)
 end
 
+desc "Automatically bumps version, edit changelog, and create pull request"
+lane :automatic_bump do |options|
+  next_version, type_of_bump = determine_next_version_using_labels(
+    repo_name: repo_name,
+    github_rate_limit: options[:github_rate_limit]
+  )
+  options[:next_version] = next_version
+  options[:automatic_release] = true
+  UI.user_error!('Skipping automatic bump since the next version is a major release') if type_of_bump == :major
```

Comment:
Hmm not sure if we did this before but I still think these shouldn't be errors but rather just messages (and skip the bump)... In this case, I'm less concerned since this could actually help detect when the automatic bump fails and we need to release the major manually

Reply:
oops yes, I think I didn't update this. Let me look into what we ended up doing in ios